Doc/tools/version_next.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+"""
+Replace `.. versionchanged:: next` lines in docs files by the given version.
+
+Run this at release time to replace `next` with the just-released version
+in the sources.
+
+No backups are made; add/commit to Git before running the script.
+
+Applies to all the VersionChange directives. For deprecated-removed, only
+handle the first argument (deprecation version, not the removal version).
+
+"""
+
+import re
+import sys
+import argparse
+from pathlib import Path
+
+DIRECTIVE_RE = re.compile(
+    r'''
+        (?P<before>
+            \s*\.\.\s+
+            (version(added|changed|removed)|deprecated(-removed)?)
+            \s*::\s*
+        )
+        next
+        (?P<after>
+            .*
+        )
+    ''',
+    re.VERBOSE | re.DOTALL,
+)
+
+docdir = (Path(__file__)
+          .parent  # cpython/Doc/tools
+          .parent  # cpython/Doc
+          .resolve()
+          )
+
+parser = argparse.ArgumentParser(
+    description=__doc__,
+    formatter_class=argparse.RawDescriptionHelpFormatter)
+parser.add_argument('version',
+                    help='String to replace "next" with.')
+parser.add_argument('directory', type=Path, nargs='?',
+                    help=f'Directory to process. Default: {docdir}',
+                    default=docdir)
+parser.add_argument('--verbose', '-v', action='count', default=0)
```

Comment:
Add a help to say it can be used several times? For example pytest says "Increase verbosity" and pip says "Give more output. Option is additive, and can be used up to 3 times."